### PR TITLE
Use NativeEventEmitter to send meeting and auth events to RN

### PR DESCRIPTION
**Pull request by jcamins**

### CODE REVIEW

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java [hunk 1]
```diff
@@ -258,9 +273,24 @@ private void unregisterListener() {
...
+  public void onShareActiveUser(long userId) {
+    if (userId == ZoomSDK.getInstance().getInMeetingService().getMyUserID()) {
+      sendEvent("MeetingEvent", "screenShareStarted");
+    } else {
+      sendEvent("MeetingEvent", "screenShareStopped");
+    }
+  }
+
+  @Override
+  public void onShareUserReceivingStatus(long l) {}
```
Comment (Zlass) [inline android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:301]: Are we overriding this to remove it's current behavior?

File: ios/RNZoomUs.m [hunk 2]
```diff
@@ -243,6 +247,19 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
...
+
+- (void)onSinkMeetingActiveShare:(NSUInteger)userID {
+  MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+  if (ms) {
+    if (userID == 0) {
+      [self sendEventWithName:@"MeetingEvent" event:@"screenShareStopped"];
+    } else if (userID == [ms myselfUserID]){
+      [self sendEventWithName:@"MeetingEvent" event:@"screenShareStarted"];
+    }
+  }
```
Comment (Zlass) [inline ios/RNZoomUs.m:264]: Is there any issue if the we try to stop a meeting that hasn't been started? Or does the `if(ms)` protect us from this?

**Review by Zlass — APPROVED** [review]

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java (hunk 1 above)
Comment (jcamins) [reply android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:301]: This is a required member of the interface, but we don't need an event for it (if we had a custom UI, we would be using this to update the name that showed next to shared content).

Comment (Zlass) [reply android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:301]: Gotcha. Just wanted to make sure this was intended and not an oops I for got to delete this.

File: ios/RNZoomUs.m (hunk 2 above)
Comment (jcamins) [reply ios/RNZoomUs.m:264]: This should only get triggered if there is a meeting underway. However, I am not entirely confident of that, so I am checking to make sure that we can access the meeting service before trying to do anything with it.

Comment (Zlass) [reply ios/RNZoomUs.m:264]: That'd make sense. I'm guessing you probably wouldn't be presented an option to trigger sharing if it wasn't ready. Or I at least hope not 😄

**Pull request merged**